process/accountProcessor.go

```diff
@@ -38,6 +38,16 @@ func NewAccountProcessor(proc Processor, pubKeyConverter core.PubkeyConverter, c
 	}, nil
 }
 
+// GetShardForAddress resolves the request by returning the shard ID for a given address for the current proxy's configuration
+func (ap *AccountProcessor) GetShardIDForAddress(address string) (uint32, error) {
```

Comment:
There is no unit test for this, it seems. Perhaps add one? But make it explicit, with less mocks?

That is, to see assertions like `erd1... -> shard 0`

Reply:
added a unit test in which I create a shard coordinator for 2 shards and call it when calling the base processor method for computing the shard. I hardcoded 2 address which should be mapped to the correct shard => less mocks, real bech32 converter and shard coordinator